spec/requests/grades_request_spec.rb

```diff
@@ -0,0 +1,139 @@
+require 'rails_helper'
+
+RSpec.describe "Admin::Grades", type: :request do
+    let(:grade) { create(:grade) }
+    let(:grades) { create_list(:subject, 3) }
+    let!(:admin) { create(:user, :admin) }
+    let(:attr) { { number: '5' , letter: "D" } }
+    let(:wrong_attr) { { number: '5' , letter: "" } }
+    let(:invalid_id) { '998' }
+
+    describe 'GET #index' do
+      context 'with logged-in admin' do
+        before(:each) do
+          sign_in admin
+          get admin_grades_path
+          it 'index status' do
+            expect(response).to have_http_status(:success)
+          end
+          it 'render index view' do
+            is_expected.to render_template :index
+          end
+        end
+      end
+    end
+
+    describe 'GET #show' do
+      before do
+        sign_in admin
+        get admin_grade_path(id: grade.id)
+      end
+      it 'show status' do
+        expect(response).to have_http_status(:success)
+      end
+      it 'should show subject' do
+        is_expected.to render_template :show
+      end
+
```

Comment:
add spec that `@grade` variable is assigned and equal to `grade`

Reply:
Can you explain what to do?